src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -1115,8 +1116,62 @@ internal unsafe int IndexOf(string source, string value, int startIndex, int cou
             return IndexOfCore(source, value, startIndex, count, options, matchLengthPtr);
         }
 
+        internal unsafe int IndexOfCore(string source, string target, int startIndex, int count, CompareOptions options, int* matchLengthPtr)
```

Comment:
Would it make sense to fold this into `internal unsafe int IndexOf(string source, string value, int startIndex, int count, CompareOptions options, int* matchLengthPtr)`; and all call that method from other places?

Reply:
Merged